diff --git a/packages/@uppy/transloadit/src/index.js b/packages/@uppy/transloadit/src/index.js
index 6c8edb2e..7ec58d5d 100644
--- a/packages/@uppy/transloadit/src/index.js
+++ b/packages/@uppy/transloadit/src/index.js
@@ -390,7 +390,7 @@ module.exports = class Transloadit extends Plugin {
   _onAssemblyFinished (status) {
     const url = status.assembly_ssl_url
     this.client.getAssemblyStatus(url).then((finalStatus) => {
-      const assemblyId = finalStatus.assemblyId
+      const assemblyId = finalStatus.assembly_id
       const state = this.getPluginState()
       this.setPluginState({
         assemblies: {
