diff --git a/src/app/containers/MediaPlayer/helpers/logMissingBlockId.js b/src/app/containers/MediaPlayer/helpers/logMissingMediaId.js
similarity index 63%
rename from src/app/containers/MediaPlayer/helpers/logMissingBlockId.js
rename to src/app/containers/MediaPlayer/helpers/logMissingMediaId.js
index 1bea27707..a25abf8bb 100644
--- a/src/app/containers/MediaPlayer/helpers/logMissingBlockId.js
+++ b/src/app/containers/MediaPlayer/helpers/logMissingMediaId.js
@@ -3,10 +3,10 @@ import { MEDIA_MISSING_FIELD } from '#lib/logger.const';
 
 const logger = nodeLogger(__filename);
 
-const logMissingBlockId = ({ url, assetType }) => {
-  const blockIdError = 'Missing Block ID';
+const logMissingMediaId = ({ url, assetType }) => {
+  const blockIdError = 'Missing Version or Block ID';
   const message = { url, assetType, blockIdError };
   logger.warn(MEDIA_MISSING_FIELD, message);
 };
 
-export default logMissingBlockId;
+export default logMissingMediaId;
diff --git a/src/app/containers/MediaPlayer/index.jsx b/src/app/containers/MediaPlayer/index.jsx
index 0ea22daa8..731344239 100644
--- a/src/app/containers/MediaPlayer/index.jsx
+++ b/src/app/containers/MediaPlayer/index.jsx
@@ -28,7 +28,7 @@ import {
   emptyBlockArrayDefaultProps,
 } from '#models/propTypes';
 import logEmbedSourceStatus from './helpers/logEmbedSourceStatus';
-import logMissingBlockId from './helpers/logMissingBlockId';
+import logMissingMediaId from './helpers/logMissingMediaId';
 
 const { logMediaPlayerStatus } = toggles[
   process.env.SIMORGH_APP_ENV || 'local'
@@ -132,8 +132,8 @@ const MediaPlayerContainer = ({
 
   const mediaIsValid = available && (versionId || blockId);
   if (!mediaIsValid) {
-    if (!blockId) {
-      logMissingBlockId({ url: assetId, assetType });
+    if (!(versionId || blockId)) {
+      logMissingMediaId({ url: assetId, assetType });
     }
     return (
       <StyledMessageContainer>
diff --git a/src/app/containers/MediaPlayer/index.test.jsx b/src/app/containers/MediaPlayer/index.test.jsx
index aa69280f9..5b4ff6a11 100644
--- a/src/app/containers/MediaPlayer/index.test.jsx
+++ b/src/app/containers/MediaPlayer/index.test.jsx
@@ -10,12 +10,12 @@ import {
   VideoAmpWithCaption,
 } from './fixtureData';
 import logEmbedSourceStatus from './helpers/logEmbedSourceStatus';
-import logMissingBlockId from './helpers/logMissingBlockId';
+import logMissingMediaId from './helpers/logMissingMediaId';
 import defaultToggles from '#lib/config/toggles';
 import onClient from '#lib/utilities/onClient';
 
 jest.mock('./helpers/logEmbedSourceStatus');
-jest.mock('./helpers/logMissingBlockId');
+jest.mock('./helpers/llogMissingMediaId');
 jest.mock('#lib/utilities/onClient');
 
 describe('MediaPlayer', () => {
@@ -59,7 +59,7 @@ it('should render the iframe when showPlaceholder is set to false', () => {
 it('should render the Media Message when there is no versionId', () => {
   const { getByText } = render(VideoCanonicalNoVersionId);
   const mediaMessage = `This content is no longer available`;
-  expect(logMissingBlockId).toHaveBeenCalledTimes(1);
+  expect(logMissingMediaId).toHaveBeenCalledTimes(1);
   expect(getByText(mediaMessage)).toBeInTheDocument();
 });
 
