diff --git a/src/app/routes/home/getInitialData/index.js b/src/app/routes/home/getInitialData/index.js
index 38c021bca..8a50b2e97 100644
--- a/src/app/routes/home/getInitialData/index.js
+++ b/src/app/routes/home/getInitialData/index.js
@@ -42,7 +42,12 @@ export default async ({ path, service, variant, pageType }) => {
     const pageDataPromise = fetchPageData({ path, pageType });
 
     const { json, status } = pageHasRadioSchedule
-      ? await withRadioSchedule({ pageDataPromise, service, path })
+      ? await withRadioSchedule({
+          pageDataPromise,
+          service,
+          path,
+          pageType: 'Home',
+        })
       : await pageDataPromise;
 
     return {
diff --git a/src/app/routes/idx/getInitialData/index.js b/src/app/routes/idx/getInitialData/index.js
index 6900e90f4..fe5104c8c 100644
--- a/src/app/routes/idx/getInitialData/index.js
+++ b/src/app/routes/idx/getInitialData/index.js
@@ -47,6 +47,7 @@ export default async ({ path, service, variant, pageType }) => {
           service,
           path,
           radioService: 'dari',
+          pageType: 'IDX',
         })
       : await pageDataPromise;
 
diff --git a/src/app/routes/liveRadio/getInitialData/index.js b/src/app/routes/liveRadio/getInitialData/index.js
index 53b372e6e..a434e63fc 100644
--- a/src/app/routes/liveRadio/getInitialData/index.js
+++ b/src/app/routes/liveRadio/getInitialData/index.js
@@ -41,6 +41,7 @@ export default async ({ path: pathname, pageType, service }) => {
           service,
           path: pathname,
           radioService: getRadioService({ service, pathname }),
+          pageType: 'LiveRadio',
         })
       : await pageDataPromise;
 
diff --git a/src/app/routes/onDemandRadio/getInitialData/index.js b/src/app/routes/onDemandRadio/getInitialData/index.js
index 0dff5a660..60e04706e 100644
--- a/src/app/routes/onDemandRadio/getInitialData/index.js
+++ b/src/app/routes/onDemandRadio/getInitialData/index.js
@@ -34,6 +34,7 @@ export default async ({ path: pathname, pageType, service }) => {
           service,
           path: pathname,
           radioService: getRadioService({ service, pathname }),
+          pageType: 'OnDemandRadio',
         })
       : await pageDataPromise;
 
diff --git a/src/app/routes/utils/withRadioSchedule/index.js b/src/app/routes/utils/withRadioSchedule/index.js
index bb29e8596..0cfb9c501 100644
--- a/src/app/routes/utils/withRadioSchedule/index.js
+++ b/src/app/routes/utils/withRadioSchedule/index.js
@@ -36,6 +36,7 @@ const withRadioSchedule = async ({
   service,
   path,
   radioService,
+  pageType,
 }) => {
   const { SIMORGH_APP_ENV, SIMORGH_BASE_URL } = process.env;
 
@@ -47,7 +48,10 @@ const withRadioSchedule = async ({
     baseUrl: SIMORGH_BASE_URL,
   });
 
-  logger.info(RADIO_SCHEDULE_REQUEST_RECEIVED, { url: radioScheduleUrl });
+  logger.info(RADIO_SCHEDULE_REQUEST_RECEIVED, {
+    url: radioScheduleUrl,
+    pageType,
+  });
   const radioSchedulePromise = fetchData(radioScheduleUrl);
 
   const [{ json: pageData, ...rest }, radioScheduleJson] = await Promise.all([
