diff --git a/test/endtoend/wdio.remote.conf.js b/test/endtoend/wdio.remote.conf.js
index ce5eb447..e1ec0ee5 100644
--- a/test/endtoend/wdio.remote.conf.js
+++ b/test/endtoend/wdio.remote.conf.js
@@ -24,7 +24,7 @@ exports.config = {
     { browserName: 'chrome', version: '70.0', platform: 'Windows 10' },
     { browserName: 'MicrosoftEdge', version: '14', platform: 'Windows 10' },
     { browserName: 'MicrosoftEdge', version: '17', platform: 'Windows 10' },
-    // { browserName: 'safari', version: '11.0', platform: 'macOS 10.12' },
+    { browserName: 'safari', version: '11.1', platform: 'macOS 10.13' },
     // { browserName: 'Safari', platformName: 'iOS', platformVersion: '12.2', deviceOrientation: 'portrait', deviceName: 'iPhone 8 Simulator' },
     { browserName: 'chrome', platformName: 'Android', platformVersion: '6.0', deviceOrientation: 'portrait', deviceName: 'Android Emulator' }
   ].map(createCapability),
